web/tests/typeahead_helper.test.js

```diff
@@ -482,8 +482,8 @@ test("sort_recipients pm counts", () => {
 
     assert.deepEqual(get_typeahead_result("b", linux_sub.stream_id, "Linux topic"), [
         "b_user_3@zulip.net",
-        "b_user_1@zulip.net",
         "b_user_2@zulip.net",
+        "b_user_1@zulip.net",
```

Comment:
This reordering is because we have more dms with `b_user_2` than with `b_user_1` and we now consider this count.